### PR TITLE
016 Script for requirement [GeneralResultCodes] INVALID_DATA wrong characters - PerformAudioPassThru

### PR DESCRIPTION
**Pull request by maria-mih**
016_ATF_N_PerformAudioPassThru_INVALID_DATA_Mandatory_Params_audioPassThruIcon_DYNAMIC.lua 
Testing coverage available at #643
All common functions used available at #703

@istoimenova Please review

### CODE REVIEW

**Comment by istoimenova** [comment]
@maria-mih, please update script according to clarification: "Clarification about expected resut code when invalid <audioPassThruIcon> is sent"
@maria-mih, please reference the script in TestSet.

**Comment by maria-mih** [comment]
Script needs to be updated and renamed due to clarification of expected result when invalid parameter received.

**Comment by maria-mih** [comment]
@istoimenova Script is renamed and updated from
016_ATF_P_PerformAudioPassThru_WARNINGS_Mandatory_Params_audioPassThruIcon_invalid_whitespaces_DYNAMIC.lua
to
016_ATF_P_PerformAudioPassThru_INVALID_DATA_Mandatory_Params_audioPassThruIcon_whitespaces_DYNAMIC.lua 

Please review it on commit 928d5dce212201300e3bfd870ce881e9df9560f0

**Comment by maria-mih** [comment]
Script is to be reworked to cover several possible scenarios with different preconditions but one expected result when request sent for mandatory parameters and image type DYNAMIC and value parameter in image structure is invalid.
It will also be included as negative check of requirement [PerformAudioPassThru] SDL must transfer request to HMI in case of valid "audioPassThruIcon" param

**Comment by maria-mih** [comment]
@istoimenova Please note that script is renamed to
016_ATF_N_PerformAudioPassThru_INVALID_DATA_Mandatory_Params_audioPassThruIcon_DYNAMIC.lua 
from 
016_ATF_P_PerformAudioPassThru_INVALID_DATA_Mandatory_Params_audioPassThruIcon_whitespaces_DYNAMIC.lua 

Script is to be reworked to cover several possible scenarios with different preconditions but one expected result when request sent for mandatory parameters and image type DYNAMIC and value parameter in image structure is invalid.
It will also be included as negative check of requirement [PerformAudioPassThru] SDL must transfer request to HMI in case of valid "audioPassThruIcon" param

File: test_scripts/API/PerformAudioPassThru/016_ATF_N_PerformAudioPassThru_INVALID_DATA_Mandatory_Params_audioPassThruIcon_DYNAMIC.lua [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    local CorIdPerformAudioPassThruAppParVD= self.mobileSession:SendRPC("PerformAudioPassThru",
+      {
+        samplingRate = "16KHZ",
+        maxDuration = 10001,
+        bitsPerSample = "8_BIT",
+        audioType = "PCM",
+        audioPassThruIcon =
+        {
+          value = params_invalid_data[ind].param_value,
+          imageType = "STATIC"
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/016_ATF_N_PerformAudioPassThru_INVALID_DATA_Mandatory_Params_audioPassThruIcon_DYNAMIC.lua]: @maria-mih, according to description, you should have imageType = "DYNAMIC"

**Comment by maria-mih** [comment]
@istoimenova The value is updated, please check on 6f662d4dd8e50c6b552c18e04b0d09a71d38d078

File: test_scripts/API/PerformAudioPassThru/016_ATF_N_PerformAudioPassThru_INVALID_DATA_Mandatory_Params_audioPassThruIcon_DYNAMIC.lua (hunk 1 above)
Comment (maria-mih) [inline test_scripts/API/PerformAudioPassThru/016_ATF_N_PerformAudioPassThru_INVALID_DATA_Mandatory_Params_audioPassThruIcon_DYNAMIC.lua]: @istoimenova Accepted and changed on 6f662d4dd8e50c6b552c18e04b0d09a71d38d078

Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/016_ATF_N_PerformAudioPassThru_INVALID_DATA_Mandatory_Params_audioPassThruIcon_DYNAMIC.lua]: @maria-mih, accepted.

**Pull request merged**